frontend/src/Signin.js

```diff
@@ -0,0 +1,59 @@
+import styles from './auth.module.css';
+import classNames from 'classnames';
+import axios from 'axios';
+import {useState} from 'react';
+
+function SignIn() {
+  const [userEmail, setUserEmail] = useState (''); 
+  const [userPassword, setUserPassword] = useState ('');
+
+  const signIn = async (_event) => { // called when LOGIN button is clicked
```

Comment:
let's practice "self documenting code" and rename this to be clear enough to no longer need the comment!

rename from `signIn` to `handleSigninSubmit` or something similar